platform-sdk/swirlds-config-extensions/src/main/java/module-info.java

```diff
@@ -7,6 +7,9 @@
 
     requires transitive com.swirlds.config.api;
     requires com.swirlds.base;
+    requires com.fasterxml.jackson.core;
+    requires com.fasterxml.jackson.databind;
     requires org.apache.logging.log4j;
+    requires org.yaml.snakeyaml;
```

Comment:
Minor: I love the `snakeyaml` because it's a simple library. However, I also see that we now have a dependency on a heavier and more advanced Jackson parser which does support YAML. While I'm not exactly a fan of the Jackson, I do recognize that it's a more powerful and versatile parser, although it suffers from security bugs quite often, sadly.

Regardless of personal preferences though, I'm wondering if it would be a good idea to use just a single parsing library, and since the Jackson is already used elsewhere in code (e.g. we're adding it at https://github.com/hashgraph/hedera-services/pull/17010 ), I'm wondering if we should use it for parsing YAML as well?

Reply:
Good point. I do not see that as a blocker but agree that we should not have 2 different serializer dependecies. We should check that. @timo0 can you create an issue?

Reply:
https://github.com/hashgraph/hedera-services/issues/17134